src/injected/visualization/focus-indicator-creator.ts

```diff
@@ -1,7 +1,6 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
-import { Point } from 'electron';
```

Comment:
There's zero reason for a connection to Electron here. The [equivalent unit test definition of Point](https://github.com/microsoft/accessibility-insights-web/blob/main/src/tests/unit/tests/injected/visualization/focus-indicator-creator.test.ts#L15) uses the definition in `injected/visualization/point`, so that's what I used here.

As an aside, this class is not used anywhere in the product, but since it's not related to Android, I'm making no attempt to remove it here.